CODE_OF_CONDUCT.md

```diff
@@ -114,12 +114,19 @@ the community.
 
 ## Attribution
 
-This Code of Conduct is adapted from the Contributor Covenant homepage, 
-version 2.0, available at
-[v2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct.html).
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.1, available at
+[https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].
 
-Community Impact Guidelines were inspired by Mozilla's code of conduct enforcement ladder.
+Community Impact Guidelines were inspired by
+[Mozilla's code of conduct enforcement ladder][Mozilla CoC].
```

Comment:
But links should be []() like this ?

Reply:
That should be <[]()> like this right?

Reply:
Links can also be like that in Markdown. It can help in scenarios where the same link is used multiple times. 🙂

Reply:
Oh :)